schemas/com.google.tag-manager.server-side/begin_checkout/jsonschema/1-0-0

```diff
@@ -0,0 +1,134 @@
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "item_category3": {
+                                                "description": "The third category hierarchy or additional taxonomy for the item.",
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "item_category4": {
+                                                "description": "The fourth category hierarchy or additional taxonomy for the item.",
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "item_category5": {
+                                                "description": "The fifth category hierarchy or additional taxonomy for the item.",
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "item_list_id": {
+                                                "description": "The ID of the list in which the item was presented to the user.",
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "item_list_name": {
+                                                "description": "The name of the list in which the item was presented to the user.",
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "item_variant": {
+                                                "description": "The item variant or unique code or description for additional item details or options.",
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "location_id": {
+                                                "description": "The location associated with the item.",
+                                                "type": ["string", "null"],
+                                                "maxLength": 4096
+                                        },
+                                        "price": {
```

Comment:
I think we can assume it's always greater than 0.

Reply:
Along with prices changing to positive, other properties that also changed for similar logic were:

- shipping and tax for `purchase` and `refund` event
- value for all events encountered (`add_payment_info`, `add_to_cart`, `add_to_wishlist`, `begin_checkout`, `generate_lead`, `purchase`, `refund`, `view_item`